.python-version

```diff
@@ -0,0 +1 @@
+3.6.8
```

Comment:
This is the minimum version this library should support. It matches IIB's https://github.com/release-engineering/iib/blob/master/.python-version